### PR TITLE
PyDocStyle fixes

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Fixes some documentation issues that arose from using the PyDocStyle pre-commit hook. When this is ready, the hook will be commented out so we don't get any errors in the workflows. 

More of these PRs will come as I fix PyDocStyle errors.

This issue addresses #370

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -44,4 +44,13 @@ repos:
...
+    - id: check-github-workflows
+
+# To be activated after quick dev cycles
+#
+- repo: https://github.com/pycqa/pydocstyle
+  rev: 6.1.1
+  hooks:
+  - id: pydocstyle
+    additional_dependencies: [toml]
+    exclude: "tests/"
```
Comment (MaxJPRey) [inline .pre-commit-config.yaml:56]: Just to not forget to comment it in order to not impact the other PRs while refactoring is not done.
```suggestion
# - repo: https://github.com/pycqa/pydocstyle
#   rev: 6.1.1
#   hooks:
#   - id: pydocstyle
#     additional_dependencies: [toml]
#     exclude: "tests/"
```

File: src/ansys/meshing/prime/core/wrappercontrol.py [hunk 2]
```diff
@@ -6,19 +7,39 @@
...
+    model : CommunicationManager
+        Server model where to create and modify WrapperControls from.
+    id : int
+        ID of the control.
+    object_id : int
+        Object ID of the control.
+    name : str
+        Name of the WrapperControl
+    local : bool, optional
+        Unused, by default False
```
Comment (MaxJPRey) [inline src/ansys/meshing/prime/core/wrappercontrol.py:23]:
```suggestion
```

**Comment by ninad-kamat** [comment]
@srpyprime Could you please review the changes in this PR to documentation.

**Comment by AlejandroFernandezLuces** [comment]
Any update on this? @srpyprime

**Comment by ninad-kamat** [comment]
@AlejandroFernandezLuces I see the code style step failing. Is this because of the new pre-commit step? If so, can you fix the failures?

**Comment by AlejandroFernandezLuces** [comment]
> @AlejandroFernandezLuces I see the code style step failing. Is this because of the new pre-commit step? If so, can you fix the failures?

We just have to revert the changes in the action. We will have to comment and uncomment the new action in order to fix the issues it highlights in several PRs. We do this to avoid having super large PRs, as we will need to modify almost all files from the project to fix the doc style issues.

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**